docs/modules/ROOT/pages/documentation/ecosystem/maven-bom.adoc

```diff
@@ -15,7 +15,7 @@ Payara's Bill Of Material (BOM) artifact collects all of these versions for the
 == Quick Start
 
 To make use of the BOM, you need to list the artifact in `dependencyManagement` section of your project in scope `import`.
-Some of the artifacts are Payara-specific patch versions which are available in our repository https://github.com/payara/Payara_PatchedProjects/[Payara_PatchedProjects].
+Some of the artifacts are Payara-specific patch versions which are available in our nexus repository https://nexus.payara.fish/[Payara_PatchedProjects].
```

Comment:
This is the full URL to the repository and the repository is now called payara-artifacts:
```suggestion
Some of the artifacts are Payara-specific patch versions which are available in our nexus repository https://nexus.payara.fish/repository/payara-artifacts/[payara-artifacts].
```